src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -359,9 +359,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
 
   <ItemGroup>
-    <RuntimeHostConfigurationOption Include=" System.Runtime.InteropServices.Marshal.IsBuiltInComSupported"
-                                    Condition="'$(BuiltInComSupport)' != ''"
-                                    Value="$(BuiltInComSupport)"
+    <RuntimeHostConfigurationOption Include="Internal.Runtime.InteropServices.ComponentActivator.IsSupported"
+                                    Condition="'$(_EnableCallingManagedFunctionFromNativeHosting)' != ''"
+                                    Value="$(_EnableCallingManagedFunctionFromNativeHosting)"
+                                    Trim="true" />
+
+    <RuntimeHostConfigurationOption Include="Internal.Runtime.InteropServices.InMemoryAssemblyLoader.IsSupported"
```

Comment:
Why does this app context switch start with `Internal.`, but the MSBuild property doesn't start with an underscore?

If this is a public property we want to advertise and document, it shouldn't be named `Internal.`. We should name it something along the lines of `System.XXX.InMemoryAssemblyLoader.IsSupported`.

Even better, the naming between the app context switch and the MSBuild property should be aligned. One is called `InMemoryAssemblyLoader` while the other is `EnableCPlusPlusCLIHostActivation`. We should be using consistent names.